ONBOARD.md

```diff
@@ -0,0 +1,113 @@
+This document describes how to add new CI jobs (utilizing ci-operator) for
+OpenShift components to the OpenShift CI system. In general, two steps are
+needed:
+
+1. [Prepare configuration file](#prepare-configuration-for-component-repo),
+   describing how ci-operator should build and test your component.
+2. [Add new jobs to Prow](#add-prow-jobs), hooking ci-operator to important
+   GitHub events for your repository.
+
+## Prepare configuration for component repo
+
+The JSON configuration file describes how to build different images in a
+testing pipeline for your repository. ci-operator has different *”targets”*
+that describe the goal images to build, and later targets build on successfully
+built earlier targets.
+
+### Source code image target
+
+By default, ci-operator builds the `src` target image. The `src` image is
+expected to contain the component source code together with all build
```

Comment:
You could mention that the code that is cloned will correctly contain the refs under test, whether that is a branch or a PR or a batch. You could also link to the `clonerefs` tool: https://github.com/kubernetes/test-infra/tree/master/prow/cmd/clonerefs